examples/dax/cluster.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: dax.aws.crossplane.io/v1alpha1
+kind: Cluster
+metadata:
+  name: example-dax-cluster
```

Comment:
think for e2e tests it would be better to use only the ref variant - what do you think ?

Reply:
Think this is a duplicate :)